### PR TITLE
enable fatality with EQ on vector population

### PR DESCRIPTION
**Pull request by Gustry**
### What does it fix ?
* Ticket : #4093 
* Funded by : DFAT
* Description : Add estimated fatality with EQ on vector pop

WIP

### CODE REVIEW

**Comment by samnawi** [comment]
Hi @Gustry I got the error message when I try your PR
image

**Comment by Gustry** [comment]
Hi @samnawi,
Sorry, it was not ready for testing. I put WIP (Work In Progress) in my description. I should have removed the label 'Needs review'. I was just using Travis to test the PR.

**Comment by samnawi** [comment]
OK @Gustry Because I see the Needs Review label in this PR.

**Comment by Gustry** [comment]
This label is put automatically by waffle when we create a PR. I need to manually remove it after PR creation.

**Comment by timlinux** [comment]
@gustry can you revisit this please for 4.1?

**Comment by Gustry** [comment]
This PR is ready for review. I can add a test later today. But @myarjunar needs this PR merged to work on the reporting side.

File: safe/definitions/post_processors.py [hunk 1]
```diff
@@ -272,6 +315,81 @@ def post_processor_population_displacement_function(
...
+        'fatality_ratio': {
+            'type': field_input_type,
+            'value': population_fatality_ratio_field
+        },
+    },
+    'output': {
+        'fatalities': {
+            'value': fatalities_field,
+            'type': formula_process,
+            'formula': 'population * fatality_ratio / 100'
```
Comment (ismailsunni) [inline safe/definitions/post_processors.py]: Why is this divided by 100? sIs fatality ratio using percent? not 0.0 - 1.0 like the other ratio?

File: safe/gui/tools/help/definitions_help.py [hunk 2]
```diff
@@ -1072,7 +1072,7 @@ def definition_to_message(
 
             if inasafe_class.get('fatality_rate') >= 0:
                 rate = inasafe_class['fatality_rate'] * 100
-                rate = u'%s%%' % rate
+                rate = u'%s%%' % format(rate, 'f')
```
Comment (lucernae) [inline safe/gui/tools/help/definitions_help.py]: Just a suggestion, perhaps you want to change it to:

```
'{:.0%}'.format(rate)
```

To make it similar with other formatting style.

P.S.: Rate doesn't have to be multiplied by 100, in this case. So, rate = 0.5 will yield 50%.

**Comment by Gustry** [comment]
Clsing this one, new PR in #4186

**Pull request closed without merging**